tailwind.config.js

```diff
@@ -8,6 +8,36 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        primary: '#1799de',
```

Comment:
**issue**: can we make this `'var(--way-color-primary)'` etc.? This way it's in sync with the defaults of the `@w2wds/core` package. Should work because we include `w2wds/core.css`.

Reply:
Not sure what you mean? you mean:

`primary: var(--way-color-primary)`

Reply:
Dont think thats possible in de config?

Reply:
mmh maybe this works : primary: 'var(--way-color-primary)',

Reply:
No as a string, which will then be used in CSS which translated to the CSS var